commit 6ff1cb355e628f8fc55fa2d01e269e5e1bbc2fe9
Author: Akinobu Mita <akinobu.mita@gmail.com>
Date:   Fri Dec 8 02:39:43 2006 -0800

    [PATCH] fault-injection capabilities infrastructure
    
    This patch provides base functions implement to fault-injection
    capabilities.
    
    - The function should_fail() is taken from failmalloc-1.0
      (http://www.nongnu.org/failmalloc/)
    
    [akpm@osdl.org: cleanups, comments, add __init]
    Cc: <okuji@enbug.org>
    Signed-off-by: Akinobu Mita <akinobu.mita@gmail.com>
    Signed-off-by: Don Mullis <dwm@meer.net>
    Signed-off-by: Andrew Morton <akpm@osdl.org>
    Signed-off-by: Linus Torvalds <torvalds@osdl.org>

diff --git a/lib/Makefile b/lib/Makefile
index 6f675c6149a7..2d6106af53cd 100644
--- a/lib/Makefile
+++ b/lib/Makefile
@@ -55,6 +55,7 @@ obj-$(CONFIG_SMP) += percpu_counter.o
 obj-$(CONFIG_AUDIT_GENERIC) += audit.o
 
 obj-$(CONFIG_SWIOTLB) += swiotlb.o
+obj-$(CONFIG_FAULT_INJECTION) += fault-inject.o
 
 lib-$(CONFIG_GENERIC_BUG) += bug.o